spec/preprocessor/preprocessor_spec.rb

```diff
@@ -55,6 +55,20 @@ def get_expected(hash, file, new_path)
     end
   end
 
+  context 'source maps' do
```

Comment:
Could you potentially add some tests that fail, and assert the expected failure cases?